d3a_api_client/redis_client_base.py

```diff
@@ -44,12 +44,13 @@ def __init__(self, area_id, client_id, autoregister=True, redis_url='redis://loc
         # TODO: Replace area_id (which is a area name slug now) with "area_uuid"
         self.area_id = area_id
         self.client_id = client_id
+        self.device_uuid = None
         self.is_active = False
         self._blocking_command_responses = {}
         self._subscribe_to_response_channels()
         self.executor = ThreadPoolExecutor(max_workers=MAX_WORKER_THREADS)
         if autoregister:
-            self.register(is_blocking=False)
+            self.register(is_blocking=True)
```

Comment:
I wonder why this was set to False before. Is this change braking the "normal" redis device client?
No need to check before the deploy. But would be great afterwards, thanks!

Reply:
This is in response to your comment [here](https://github.com/gridsingularity/d3a-api-client/pull/40#discussion_r449414220), now I had to explicitly wait for response to get device_uuid from d3a which is necessary for mapping aggregator with device based on their uuids.